test/source/tests/page-recipe/compose-page-recipe.ts

```diff
@@ -111,7 +111,7 @@ export class ComposePageRecipe extends PageRecipe {
         if (sendingType !== 'to') { // input-to is always visible
           await composePageOrFrame.waitAndClick(`@action-show-${sendingType}`);
         }
-        await composePageOrFrame.waitAndType(`@input-${sendingType}`, email);
+        await composePageOrFrame.waitAndType(`@input-${sendingType}`, email + '\n');
```

Comment:
Seems like this also fixed https://github.com/FlowCrypt/flowcrypt-browser/issues/4593
Let's see if we see new same semaphoreci test after this merge